CrewChiefV4/SpeechRecogniser.cs

```diff
@@ -251,6 +272,25 @@ public void initialiseSpeechEngine()
                 validateAndAdd(WHATS_MY_FUEL_LEVEL, staticSpeechChoices);
                 validateAndAdd(WHATS_MY_FUEL_USAGE, staticSpeechChoices);
 
+                foreach (String s in CALCULATE_FUEL_FOR)
+                {
+                    if (s == null || s.Trim().Count() == 0)
+                    {
+                        continue;
+                    }
+                    foreach (KeyValuePair<String, int> entry in numberToNumber)
+                    {
+                        if(entry.Value == 1)
+                        {
+                            staticSpeechChoices.Add(s + " " + entry.Key + " " + "lap");
+                        }
+                        else
+                        {
+                            staticSpeechChoices.Add(s + " " + entry.Key + " " + "laps");
```

Comment:
again, loop over the LAPS array instead of hard coding "laps" (yeah, I know LAPS only contains "laps" but it might grow)